book.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training" />
+    <meta name="author" content="Wolox" />
+    <link rel="stylesheet" type="text/css" href="application.css" />
+  </head>
+  <body class="book">
```

Comment:
don't add classes to body tag

Reply:
Also, maybe the class `book` would be inside the `book-container` :stuck_out_tongue_closed_eyes: